src/assets/styles/_align.scss

```diff
@@ -0,0 +1,20 @@
+body, html {
+  position: relative;
+  overflow: hidden;
+}
+
+  .v-mid,.v-m {
```

Comment:
Remove this extra space from each file wherever it has occurred and ensure proper indentation.